pkg/apis/fission.io/v1/tool/swagger_type_docs.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// Please refer https://github.com/kubernetes/kubernetes/blob/master/cmd/genswaggertypedocs/swagger_type_docs.go for original file
+//
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	kruntime "k8s.io/apimachinery/pkg/runtime"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/klog"
```

Comment:
Why are we adding this?

Reply:
This tool parses all struct fields and its command for go-restful. So that we don't need to write the go-restful description manually.